src/app-layout/__tests__/common.test.tsx

```diff
@@ -248,54 +240,38 @@ describeEachAppLayout(size => {
   describe(`Drawers`, () => {
     test(`Should call handler once on open when toggle is clicked`, () => {
       const onChange = jest.fn();
-      const drawersClosed = {
-        drawers: {
-          onChange: onChange,
-          items: singleDrawer.drawers.items,
-        },
-      };
-      const { wrapper } = renderComponent(<AppLayout contentType="form" {...(drawersClosed as any)} />);
```

Comment:
Here and below, I remove `contentType="form"` because it is not needed for the actual tests. It was there to fix a type error when spreading props of type `any `.

Not needed anymore.